style.css

```diff
@@ -540,3 +544,221 @@ span {
   font-size: 13px;
   font-weight: 600;
 }
+
+.internal-link {
+  display: none;
+}
+
+@media screen and (min-width: 1024px) {
+  .contact textarea {
+    width: 309px;
+    height: 100px;
+  }
+
+  .ending-bar { display: none; }
+
+  .wrap .contact button {
+    align-self: center;
+    padding: 10px;
+  }
+
+  .contact {
+    justify-content: center;
+    align-items: center;
+  }
+
+  .wrap .contact-para { margin-bottom: 20px; }
+
+  .contact-title { margin-bottom: 0; }
+
+  #navigator_bar {
+    margin-top: 0;
+    margin-right: 5%;
+    margin-left: 10.76%;
+    width: 100%;
+  }
+
+  header #navigator_bar .logo {
+    margin-right: 30%;
+    font-family: poppins, sans-serif;
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  #navigator_bar nav {
+    display: none;
+  }
+
+  #navigator_bar .internal-link {
+    display: flex;
+    justify-content: space-between;
+    min-width: 190px;
+    margin-top: 30px;
+    line-height: 20px;
+  }
+
+  #navigator_bar .internal-link li {
+    list-style: none;
+  }
+
+  #navigator_bar .internal-link li a {
+    text-decoration: none;
+    font-family: poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .starting-page {
+    height: 100vh;
+    background-image: url("assets/Group\ 58.png");
+    background-repeat: no-repeat;
+    background-size: 90%;
+  }
+
+  #section-presentation {
+    height: 916px;
+    margin-left: 18%;
+    margin-right: 18%;
+    margin-top: 200px;
+    justify-content: stretch;
+    background-image: none;
+  }
+
+  #my_work {
+    margin-top: 100px;
+    display: grid;
+    grid-template-rows: repeat(18, 100vh);
+  }
+
+  .general {
+    grid-column: 2 / 1 span;
+    align-self: center;
+  }
+
+  .project {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    width: 80%;
+    height: 90%;
+    margin-left: 10%;
+    grid-column-gap: 20px;
+    margin-bottom: 100px;
+  }
+
+  #general_inf {
+    grid-column: 2 / 1 span;
+  }
+
+  .project footer .external_link {
+    margin-left: 0;
+    width: 18%;
+    padding: 2% 0;
+  }
+
+  .project_image {
+    grid-column: 1 / 1 span;
+
+    /* max-width: 100%;
+    min-height: 70%;
+    max-height: 411px; */
```

Comment:
- [ ] Kindly remove unwanted commented code from your base code because they will make your project look messy. If you need to review a previous version of your code, you can always use git history. :slightly_smiling_face:

Reply:
Thank you for notifying me. I have removed the comment portion in the file.